packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -21,43 +21,54 @@ interface GasEstimatorMapping {
     backupUrl?: string;
   };
 }
-
-interface EtherchainResponse {
-  safeLow: number;
-  standard: number;
+interface EthGasStationResponse {
   fast: number;
   fastest: number;
-  currentBaseFee: number;
-  recommendedBaseFee: number;
+  safeLow: number;
+  average: number;
+  block_time: number;
+  blockNum: number;
+  speed: number;
+  safeLowWait: number;
+  avgWait: number;
+  fastWait: number;
+  fastestWait: number;
+  gasPriceRange: Record<string, number>;
 }
 
+interface MaticGasPriceData {
+  maxPriorityFee: number | string;
+  maxFee: number | string;
+}
 interface MaticResponse {
-  safeLow: number;
-  standard: number;
-  fast: number;
-  fastest: number;
+  safeLow: MaticGasPriceData;
+  standard: MaticGasPriceData;
+  fast: MaticGasPriceData;
+  estimatedBaseFee: string;
   blockTime: number;
   blockNumber: number;
 }
 
 export const MAPPING_BY_NETWORK: GasEstimatorMapping = {
-  // Expected shape:
-  // <netId>: {
-  //     url: <primary-gas-station-url>,
-  //     backupUrl: <optional-backup-gas-station-url>,
-  //     defaultFastPricesGwei: <default-gas-price-for-network>
-  // }
   1: {
-    url: "https://www.etherchain.org/api/gasPriceOracle",
-    defaultMaxFeePerGasGwei: 500,
-    defaultMaxPriorityFeePerGasGwei: 5,
+    url: "https://ethgasstation.info/api/ethgasAPI.json",
+    defaultMaxFeePerGasGwei: 450, // maxFeePerGas = baseFeePerGas + maxPriorityFeePerGas
+    defaultMaxPriorityFeePerGasGwei: 150,
     type: NetworkType.London,
   },
   10: { defaultFastPriceGwei: 1, type: NetworkType.Legacy },
-  137: { url: "https://gasstation-mainnet.matic.network", defaultFastPriceGwei: 10, type: NetworkType.Legacy },
+  137: {
+    url: "https://gasstation.polygon.technology/v2",
+    defaultFastPriceGwei: 250,
```

Comment:
The previous endpoint `https://gasstation-mainnet.matic.network` doesn't work anymore
Default fast price in matic was very low

Reply:
Why do we still use Legacy type for Polygon if gasstation.polygon.technology returns London type fees (maxPriorityFee and maxFee)?

Reply:
Thanks @Reinis-FRP 
You are totally right,  we can send [eip1559-transactions in polygon](https://wiki.polygon.technology/docs/develop/eip1559-transactions/how-to-send-eip1559-transactions#sending-eip1559-transactions)

Fixed